app/templates/content/about-us.html

```diff
@@ -7,37 +7,50 @@
-  <p><strong>We're transforming how government buys digital and technology services.</strong></p>
-
-  <h2>Our purpose</h2>
-  <p>
-    The Digital Marketplace is an initiative under the National Innovation and Science Agenda. Our purpose is to make it easier for smaller businesses to compete for the government’s $6 billion a year spend on information and communications technology (ICT) products and services.
-  </p>
-  <p>
-    To do this we’re building an ecosystem where government buyers and sellers can connect and have the procurement interactions they need — at the level of simplicity or complexity appropriate to their circumstances — and to make the outcomes of those interactions more transparent.
-  </p>
-  
-  <h2>Our progress</h2>
-  <p>
-    The Digital Marketplace is an agile development in Beta. This means the site will evolve over time in response to user needs. If you have any questions, feedback or would like to be involved in user research, contact us at <a href="mailto:marketplace@digital.gov.au">marketplace@digital.gov.au</a>.
-  </p>
-  
-  <h2>How we comply with the procurement rules</h2>
-  <p>
-    Sellers contracted through the Digital Marketplace are part of the Digital Marketplace Panel (SON3413842). This panel was established through an open approach to market by the Digital Transformation Agency and has complied with the requirements of conducting an open tender as set out in the Commonwealth Procurement Rules. All panel members have accepted our <a href="{{ asset_path }}media/documents/digital-marketplace-master-agreement.pdf" target="_blank">Master Agreement (PDF 229KB)</a>.
-  </p>
-  <p>
-    Although the Digital Marketplace makes it easy to comply with the Commonwealth Procurement Rules as a government buyer, you are still responsible for complying with these rules and any Accountable Authority Instructions related to procurement in your agency or issued by the Department of Finance. If you’re not sure of these requirements, please check with the procurement section in your own agency.
-  </p>
-  <p>
-    Local, state and territory government buyers are welcome to use the Digital Marketplace but should make sure that they comply with any additional procurement rules that apply in their state or territory.
-  </p>
-  
-  <h3>Government buyers</h3>
-  <p><a href="/buyers-guide">How to buy from the Digital Marketplace</a></p>
-
-  <h3>Digital professionals</h3>
-  <p><a href="/sellers-guide">How to sell through the Digital Marketplace</a></p>
-  <p><a href="/2/signup">Join the Digital Marketplace</a></p>
+  <p><strong>On this page</strong></p>
+  <ul>
+    <li><a href="#whatthisservicedoes">what this service does</a></li>
+    <li><a href="#whocanbeabuyer">who can be a buyer</a></li>
+    <li><a href="#whocanbeaseller">who can be a seller</a></li>
+    <li><a href="#adviceandsupport">advice and support</a></li>
+  </ul>
+  <h2><a name="whatthisservicedoes"></a>What this service does</h2>
```

Comment:
I think we can improve on these empty `<a>` tags.  See how the links on https://www.gov.uk/service-manual/design/making-prototypes#when-to-use-prototypes are marked up with `id`.